src/main/antlr4/no/ssb/jsontemplate/antlr4/JsonTemplateAntlr.g4

```diff
@@ -37,7 +37,7 @@ variable : '$'variableName;
 variableName : IDENTIFIER;
 RAW : '`' .*? '`';
 
-IDENTIFIER : [a-zA-Z0-9_'.']+;
+IDENTIFIER : [a-zA-Z0-9_'.]+;
```

Comment:
Is the diff here intended?
```suggestion
IDENTIFIER : [a-zA-Z0-9_'.']+;
```

Reply:
Yes, the diff is intended. After consulting with the Antlr-plugin in IntelliJ, the line of code should be:
`IDENTIFIER : [a-zA-Z0-9_'.]+;`